### PR TITLE
chore(package): tidy root package json

### PR DESCRIPTION
**Pull request by AlexBramhill**
# Issue
There are non-dev dependancies on the root of the repo

# Resolution
I've moved these to devDependancies

# Todo
Update `package.lock` once I can successfully `npm run ci:all`

### CODE REVIEW

**Comment by abhi-markan** [comment]
Please adjust the commit title so it adheres to the standards.

**Comment by lmarrai-sw** [comment]
@AlexBramhill I think we need to edit the lint config to tell it that any dependencies in the `e2e-tests` folder are only dev dependencies

File: package.json [hunk 1]
```diff
@@ -31,29 +31,28 @@
...
-    "@commitlint/config-conventional": "^17.7.0",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "cspell": "^6.31.3",
     "eslint": "^8.50.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-cypress": "^2.15.1",
     "eslint-plugin-import": "^2.28.1",
     "husky": "^8.0.3",
     "lint-staged": "^13.3.0",
+    "moment": "^2.29.4",
```
Comment (AlexBramhill) [inline package.json:54]: I don't know where this has suddenly appeared from and will delete

File: package.json [hunk 2]
```diff
@@ -32,29 +32,28 @@
       "cspell lint --gitignore --no-must-find-files --unique --no-progress --show-suggestions --color"
     ]
   },
-  "dependencies": {
+  "dependencies": {},
```
Comment (lmarrai-sw) [inline package.json:35]:
```suggestion
```

File: e2e-tests/.eslintrc.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ module.exports = {
     'import/no-named-as-default': 'off',
     'implicit-arrow-linebreak': 'off',
     'import/prefer-default-export': 'off',
-    'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
+    'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
```
Comment (lmarrai-sw) [inline e2e-tests/.eslintrc.js:24]:
```suggestion
    'import/no-extraneous-dependencies': ['error'],
```
Is `devDependencies: true` not the default?

**Review by lmarrai-sw — DISMISSED** [review]

**Review by abhi-markan — APPROVED** [review]

**Comment by abhi-markan** [comment]
@AlexBramhill merge conflicts to resolve please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Linting errors to address.

**Review by abhi-markan — DISMISSED** [review]

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**